packages/console/src/pages/Mfa/types.ts

```diff
@@ -3,8 +3,9 @@ import { type MfaPolicy, type SignInExperience } from '@logto/schemas';
 export type MfaConfig = SignInExperience['mfa'];
 
 export type MfaConfigForm = {
-  policy: MfaPolicy;
   totpEnabled: boolean;
   webAuthnEnabled: boolean;
   backupCodeEnabled: boolean;
+  isMandatory: boolean;
+  setUpPrompt: MfaPolicy;
```

Comment:
```suggestion
  setUpPrompt: Optional<Exclude<MfaPolicy, MfaPolicy.UserControlled | MfaPolicy.Mandatory>>;
```

Reply:
In this way, it is hard to handle type checking, I think `MfaPolicy` is good enough, we don't need the type checking here, we have code logic.

Reply:
Can you give an example? Your form data format utils should handle those returned data types already. Didn't see a issue here.  Better make the form data type more accurate.

Reply:
In `convertMfaConfigToForm` and `convertMfaFormToConfig`, there are several typing issues that require additional effort to resolve, and I don’t believe it’s worth the time or effort.